src/js/_enqueues/wp/updates.js

```diff
@@ -798,7 +798,7 @@
 	 */
 	wp.updates.installPluginSuccess = function( response ) {
 		var $message = $( '.plugin-card-' + response.slug + ', #plugin-information-footer' ).find( '.install-now' ),
-			buttonText = _x( 'Installed!', 'plugin' ),
+			buttonText = __( 'Installed!' ),
```

Comment:
Example of where the exclamation mark is retained, and the string below also has one

Reply:
I‘ll double check.